iModelCore/iModelPlatform/DgnCore/SnapContext.cpp

```diff
@@ -339,6 +339,7 @@ double                  m_snapAperture;
 DPoint3d                m_closePtWorld;
 DPoint3d                m_closePtLocalCorrected;
 DMap4d                  m_worldToView;
+Transform               m_geometryToWorld;

```

Comment:
What do you think of calling this "modelToWorld" everywhere? That makes it clearer to me that it's different from the placement transform...

Reply:
Yeah that's better.